m2cgen/assemblers/boosting.py

```diff
@@ -4,6 +4,8 @@
 from m2cgen.assemblers import utils
 from m2cgen.assemblers.base import ModelAssembler
 
+MAX_LEAVES_PER_METHOD = 3000
```

Comment:
I actually liked the `MAX_LEAVES_PER_SUBROUTINE` more, since method is something specific to Java when assemblers should be language independent.

Reply:
I renamed it because pylint was mad about line lengths (which are super short at 80chars) :(

Reply:
I'm fine calling it `LEAVES_CUTOFF_THRESHOLD` or ``LEAVES_CUTOFF_LIMIT` if that helps